components/emoji_picker/emoji_picker.jsx

```diff
@@ -481,7 +489,12 @@ export default class EmojiPicker extends React.PureComponent {
         }
     }
 
-    getCategoryByIndex(index) {
+    onSkinSelected = (skin) => {
+        this.props.actions.setRecentSkin(skin);
+        this.props.actions.persistRecentSkin(skin);
```

Comment:
Also, moving the user's current skin to preferences means we don't need to explicitly persist it